src/internal/packager/deploy.go

```diff
@@ -62,9 +63,31 @@ func Deploy() {
 
 	spinner.Success()
 
+	sbomViewFiles, _ := filepath.Glob(tempPath.sboms + "/sbom-viewer-*")
+	// If SBOM files exist, temporary place them in the deploy directory
+	if len(sbomViewFiles) > 0 {
+		sbomDir := "zarf-sbom"
+		// Cleanup any failed prior removals
+		_ = os.RemoveAll(sbomDir)
+		// Create the directory again
+		utils.CreateDirectory(sbomDir, 0755)
+		for _, file := range sbomViewFiles {
+			// Our file copy lib explodes on these files for some reason...
```

Comment:
This works so I guess that's fine but I'm curious as to why the copy wouldn't work.. Probably something that's worth  looking into when we have more time.

Reply:
Yeah it's a bug in that library. We actually only introduced it for the recursive copy feature we really don't use anymore so I'd like to look at moving everything to this pattern (standard lib) and drop that dep soon.